Demo/.idea/.idea.Demo/.idea/projectSettingsUpdater.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Since they are already in the repo, any delete form my side is now ignored. I guess I should make two commits. One changing the .gitignore so the idea files are visible again + deleting them. And then a commit ignoring them again. lets try.